### ISSUE TITLE
Support for postgresql 15.1

### ISSUE THREAD

**Issue by e12e** [UpCloudLtd/terraform-provider-upcloud#365]
Trying to create a new database, I get error from `terraform plan`:

```
Planning failed. Terraform encountered an error while generating this plan.

╷
│ Error: expected version to be one of [12 13 14], got 15.1
│ 
│   with upcloud_managed_database_postgresql.********,
│   on gazelle.tf line 16, in resource "upcloud_managed_database_postgresql" "*******":
│   16:     version                             = "15.1
```

```
terraform version #output:
Terraform v1.4.5
on darwin_arm64
+ provider registry.terraform.io/upcloudltd/upcloud v2.9.1
```

With version set to "14" terraform works as expected.

**Comment by kangasta** [comment]
Thank you for reporting this issue! We will update the validation to allow also `15` as value for the `version` field.

**Comment by kangasta** [comment]
Support for Postgres version 15 is now available in [v2.10.0](https://github.com/UpCloudLtd/terraform-provider-upcloud/releases/tag/v2.10.0)
